x-pack/test/performance/tests/playwright/ecommerce_dashboard.ts

```diff
@@ -5,52 +5,68 @@
  * 2.0.
  */
 import Url from 'url';
+import { Page } from 'playwright';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ecommerceDashboard({ getService }: FtrProviderContext) {
   describe('ecommerce_dashboard', () => {
-    const config = getService('config');
-    const performance = getService('performance');
-    const logger = getService('log');
+    it('ecommerce_dashboard', async () => {
+      const config = getService('config');
+      const performance = getService('performance');
+      const logger = getService('log');
 
-    const { step } = performance.makePage('ecommerce_dashboard');
+      await performance.runUserJourney(
+        'ecommerce_dashboard',
+        [
+          {
+            name: 'Go to Sample Data Page',
+            handler: async ({ page }: { page: Page }) => {
+              const kibanaUrl = Url.format({
```

Comment:
Maybe we can provide `url`  via `StepCtx` to avoid repeating this logic in every test?